include/picongpu/particles/atomicPhysics2/interfaceDefintion.hpp

```diff
@@ -0,0 +1,36 @@
+
+#include <pmacc/math/Vector.hpp>
+
+namespace picongpu::particles
+{
+    namespace compileTime = pmacc::math::CT;
+
+    // in-cell position to initialize macro particle to
+    CONST_VECTOR(
+        /*datatype*/ float_X,
+        /*dim*/ 3,
```

Comment:
should be `simDim` or is atomic physics assuming always 3 components?

Reply:
that is part of the rejected `one particle per superCell position functor` that will not go into the mainline anyway. but yes that should have probably accounted for simDim 2.